apps/docs/content/components/concepts/color-schemes.mdx

```diff
@@ -22,11 +22,11 @@ When the `system` value is provided, an additional fallback color scheme must be
 
 <Example src="HopperProvider/docs/color-scheme/apply-system" isOpen />
 
-Color schemes can also be nested. You could have a dark themed dialog inside a light themed application. For example, this might be useful for certain experiences like color scheme previews, where you want to showcase a specific color scheme, regardless of the operating system setting.
+Color schemes can also be nested. You could have a dark themed dialog inside a light themed application. This might be useful for certain experiences such as color scheme previews, where you want to showcase a specific color scheme regardless of the operating system setting.
```

Comment:
```suggestion
Color schemes can also be nested. You could have a dark themed dialog inside a light themed application. This might be useful for certain experiences, such as color scheme previews, where you want to showcase a specific color scheme regardless of the operating system's setting.
```